openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/QueryParameterTests.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.openapi.generators.openapi;
+
+import io.ballerina.openapi.converter.OpenApiConverterException;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Tests for query parameter mapping.
+ */
+public class QueryParameterTests {
+    private static final Path RES_DIR = Paths.get("src/test/resources/ballerina-to-openapi/").toAbsolutePath();
+    private Path tempDir;
+
+    @BeforeMethod
+    public void setup() throws IOException {
+        this.tempDir = Files.createTempDirectory("bal-to-openapi-test-out-" + System.nanoTime());
+    }
+    // type BasicType boolean|int|float|decimal|string ;
+    //public type  QueryParamType () |BasicType|BasicType[];
```

Comment:
Do we need this commented lines?

Reply:
I added this to make aware others who go through test scenarios of the supported data type in query parameter ballerina. Shall I add this comment more descriptively? :)

Reply:
yeah :) fixed with 5024755